tests/js/spec/stores/useLegacyStore.spec.tsx

```diff
@@ -0,0 +1,27 @@
+import {act, mountWithTheme} from 'sentry-test/reactTestingLibrary';
+
+import TeamStore from 'app/stores/teamStore';
+import {useLegacyStore} from 'app/stores/useLegacyStore';
+
+describe('abc', () => {
```

Comment:
should we update the description to something more descriptive?

Reply:
oh right